### PR TITLE
kubel-exec-pod-by-shell-command

### PR DESCRIPTION
**Pull request by TxGVNN**
Hi,

I don't know this feature is useful for everyone or not. But it helps me in some cases. Example run `ip a`, `nslookup`, some command to control something.

### CODE REVIEW

**Comment by abrochard** [comment]
I love this! Thank you for submitting. Before I merge, can you add that new command to the readme?

**Comment by TxGVNN** [comment]
> Before I merge, can you add that new command to the readme?

Ya, sure. I wonder should I remove key binding in `kubel-exec-popup`. What do you think?

**Comment by abrochard** [comment]
> Ya, sure. I wonder should I remove key binding in `kubel-exec-popup`. What do you think?

I would keep it. In general I prefer to not break existing shortcuts and they aren't really conflicting.

File: README.md [hunk 1]
```diff
@@ -60,6 +61,7 @@ enter => get resource details
 C-u enter => describe resource
 h => help popup
 ? => help popup
+! => run fast shell-command
```
Comment (abrochard) [inline README.md:64]: the wording is confusing I think. Could you rephrase to `quick run shell-command`? I think that "run fast shell-command" implies that the command is going to run fast, not that it's easy to run it

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ All notable changes to this project will be documented in this file.
 - run `kubectl describe pod`
 - `kubectl` apply a buffer to current context/namespace
 
+### Added
+- `kubel-exec-pod-by-shell-command` by using shell-command we can run fast a command
```
Comment (abrochard) [inline CHANGELOG.md:15]: same (and sorry for the nitpick) but I would say "we can quickly run a command"

File: README.md (hunk 1 above)
Comment (TxGVNN) [reply README.md]: > going to run fast, not that it's easy to run it

Haha :laughing:, you're right. I updated, thank you

**Review by abrochard — APPROVED** [review]

**Comment by abrochard** [comment]
Thank you!
